homeassistant/components/zha/entity.py

```diff
@@ -61,7 +61,6 @@ def __init__(self, unique_id: str, zha_device: ZHADevice, **kwargs: Any) -> None
         self._extra_state_attributes: dict[str, Any] = {}
         self._zha_device = zha_device
         self._unsubs: list[Callable[[], None]] = []
-        self.remove_future: asyncio.Future[Any] = asyncio.Future()
```

Comment:
This property is moved to ZhaEntity since it seems it is only used there. Else it would also initialize for `ZhaGroupEntity`.

Reply:
I think this is fine unless `ZhaGroupEntity` can be removed from the network in `core/gateway.py`

Will leave it for someone who is more familiar with zha to be sure

Reply:
Group entities don't generate remove events so this should be good. Thanks!